### PR TITLE
Support status codes 302 and 200 in GetArchiveLink endpoint

### PR DESCRIPTION
**Pull request by assembly-winston**
Fixes: #3171

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @assembly-winston !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

Also awaiting signed CLA before proceeding.

**Comment by assembly-winston** [comment]
Signed CLA!

**Comment by gmlewis** [comment]
OK, I'm going to override the normal policy of a second LGTM for this simple change (similar to adding a field to a struct) and go ahead and merge.

**Pull request merged**
